.github/workflows/aws-e2e-tests-non-root.yaml

```diff
@@ -0,0 +1,75 @@
+name: AWS E2E Tests (Non-root)
+run-name: AWS E2E Tests (Non-root) - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
+  pull_request:
+    paths:
+      - '.github/workflows/aws-e2e-tests-non-root.yaml'
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
+  merge_group:
+    paths:
+      - '.github/workflows/aws-e2e-tests-non-root.yaml'
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
+
+env:
+  TEST_KUBE: true
```

Comment:
what's the purpose of this env var? I see it only gets echo'd by the Makefile target

Reply:
Yeah, it's used to skip tests if you wanna run them locally